NuGet.Docs/Contribute/Default.cshtml

```diff
@@ -1,4 +1,13 @@
 ﻿<div class="article-boxes">
+
+    <article class="box links wide">
+	<header>
+	    <h1>NuGet Contributions</h1>
+        </header>
+	<p>NuGet is an open source project and welcomes contributions from anyone in the communiuty.  Your contributions will all be held to the same standard as those on the project team.</p>
+	<p>For a quick list of items that the NuGet team has identified for anyone in the community to resolve, check the <a href="https://github.com/NuGet/Home/issues?q=is%3Aopen+is%3Aissue+label%3A%22Up+for+Grabs%22">Up for Grabs</a> label in the issues list within the Home repository.</p>
```

Comment:
What if instead of anyone in the community we say:

has identified as good issues to look into